test/integration/patients/patient/sidebar.js

```diff
@@ -267,15 +366,25 @@ context('patient sidebar', function() {
       .routePrograms()
       .routeAllProgramActions()
       .routeAllProgramFlows()
+      .route({
+        status: 504,
+        method: 'GET',
+        url: 'api/patients/1/engagement-status',
+        response: {},
+        delay: 500,
```

Comment:
Not super ideal, but CI executes this too fast for the engagement status loading state to display.

Reply:
might be a way to do this with `cy.clock`. Is 500 really needed too? Seems high